public/js/main.js

```diff
@@ -34,40 +34,78 @@
   var copyContainer = $('<div/>');
   copyEl.attr('style', 'display: none;');
   copyEl.appendTo('body');
+  var clipboard;
 
   function setupMouseEvents() {
-    $('.library-column').on( "mouseenter", function(ev) {
-      var cont = $(ev.currentTarget);
-      copyEl.show();
-      copyEl.appendTo(cont);
-    })
-    .on( "mouseleave", function(ev) {
-      var cont = $(ev.currentTarget);
-      //copyEl.appendTo('body');
-    });
+    // Currently not showing the copy button for iOS, check clipboard.js support
+    if(!(/iPhone|iPad/i.test(navigator.userAgent))) {
+      $('.library-column').on( "mouseenter", function(ev) {
+        var cont = $(ev.currentTarget);
+        copyEl.show();
+        copyEl.appendTo(cont);
+      })
+      .on( "mouseleave", function(ev) {
+        var cont = $(ev.currentTarget);
```

Comment:
Dead code? That code registers the mouse event hover action to show the copy button. Removing would break functionality, right?